packages/components/src/dropdown-menu/index.native.js

```diff
@@ -77,7 +76,7 @@ function DropdownMenu( {
 			popoverProps={ mergedPopoverProps }
 			renderToggle={ ( { isOpen, onToggle } ) => {
 				const openOnArrowDown = ( event ) => {
-					if ( ! isOpen && event.keyCode === DOWN ) {
+					if ( ! isOpen && event.code === 'ArrowDown' ) {
```

Comment:
From researching this a bit, I believe this is unused code. It is likely safe to remove `onKeyDown` and related code entirely from this file. 

For web, the `onKeyDown` is passed along via [prop spread](https://github.com/WordPress/gutenberg/blob/bf22040dc217a6846a0379e07e350ea843660c08/packages/components/src/button/index.js#L154). For native, we [do not prop spread](https://github.com/WordPress/gutenberg/blob/bf22040dc217a6846a0379e07e350ea843660c08/packages/components/src/button/index.native.js#L177-L186) `onKeyDown`, also it is not a valid prop for React Native's [`TouchableWithoutFeedback`](https://reactnative.dev/docs/0.66/touchablewithoutfeedback). 

* `onBlur`
* `onFocus`
* `onLayout`
* `onLongPress`
* `onPress`
* `onPressIn`
* `onPressOut`

All that said, it is safe to ship this as-is. The unused code is not necessarily related to this work.

Reply:
Since I had to rebase anyway to solve conflicts, I went ahead and removed the `onKeyDown` prop in https://github.com/WordPress/gutenberg/pull/43439/commits/f461f65f29b327903a3d51c80b67ad6f4910ea20

Let me know if the change look good to you!

Reply:
Actually I noticed that some tests are failing in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5109.

I've reverted the removal of `onKeyDown`, let's see if it makes any difference

Reply:
I think some of the errors in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5109 should be solved with changes from https://github.com/WordPress/gutenberg/pull/43521, at least the ones affecting `Gutenberg Editor Slash Inserter tests should insert an embed image block with "/img" + enter` test case ([reference](https://app.circleci.com/pipelines/github/wordpress-mobile/gutenberg-mobile/18368/workflows/52857fc8-29bf-4d81-8efa-975cbac61a8c/jobs/100679/tests#failed-test-0)).

Reply:
Thank you for the context — I'll wait for it to get merged and take a look at the tests again

Reply:
https://github.com/WordPress/gutenberg/pull/43439/commits/f461f65f29b327903a3d51c80b67ad6f4910ea20 looks sound to me. I agree with @fluiddot that the CI failures are unrelated to this PR. Hopefully we validate that once https://github.com/WordPress/gutenberg/pull/43521 is merged and this branch is rebased.

Reply:
As https://github.com/WordPress/gutenberg/pull/43521 got merged, I rebased and restored the removal of the `onKeyDown` callback.

I'll wait for your final ✅  before merging (and will keep an eye for unit test failures)